data_structures/src/transaction.rs

```diff
@@ -466,14 +466,44 @@ pub struct MintTransaction {
 
 impl MintTransaction {
     /// Creates a new mint transaction.
-    pub fn new(epoch: Epoch, output: ValueTransferOutput) -> Self {
+    pub fn new(epoch: Epoch, outputs: Vec<ValueTransferOutput>) -> Self {
         MintTransaction {
             epoch,
-            output,
+            outputs,
             hash: MemoHash::new(),
         }
     }
 
+    pub fn with_split_utxos(
+        epoch: Epoch,
+        reward: u64,
+        own_pkh: PublicKeyHash,
+        collateral_minimum: u64,
+        utxo_counter: usize,
+        utxos_to_achieve: usize,
```

Comment:
Can you refactor this into only one argument? By changing the calls to `utxos_to_achieve.saturating_sub(utxo_counter)`.

```suggestion
        utxos_to_achieve: usize,
```